docs/getting-started-apm-server.asciidoc

```diff
@@ -0,0 +1,314 @@
+[[getting-started-apm-server]]
+== Getting Started With APM Server
+
+// This section was copied from the APM Overview
+// Switch to tagged regions when available
+The easiest way to get started with APM Server is by using our
+https://www.elastic.co/cloud/elasticsearch-service[hosted {es} Service] on
+Elastic Cloud. The {es} Service is available on AWS, GCP, and Azure,
+and automatically configures APM Server to work with {es} and {kib}.
+
+[float]
+=== Hosted Elasticsearch Service
+
+Skip managing your own {es}, {kib}, and APM Server by using our
+https://www.elastic.co/cloud/elasticsearch-service[hosted {es} Service] on
+Elastic Cloud.
+
+image::images/apm-architecture-cloud.png[Install Elastic APM on cloud]
+
+https://www.elastic.co/cloud/elasticsearch-service/signup[Try out the {es} Service for free],
+and then begin configuring your own user settings right in the Elasticsearch Service Console.
+Any changes are automatically appended to the `apm-server.yml` configuration file for your instance.
+A full list of configuration options is available in Elastic Cloud's
+{cloud}/ec-manage-apm-settings.html[Add APM user settings].
+
+[float]
+=== Install the stack yourself
```

Comment:
_Install_ and/or _Run_ ?

Reply:
I'm okay with changing this, but if we do, I'll need to change it in APM Overview as well.

What about "Install and manage the stack yourself"?

Reply:
SGTM

Reply:
Changed